src/Editor/Editor.php

```diff
@@ -37,21 +38,40 @@ public function register(): void
 	 */
 	public function getEditorBackLink(): void
 	{
-		$actialUrl = UtilsGeneralHelper::getCurrentUrl();
-		$postType = Forms::POST_TYPE_SLUG;
-		$page = FormAdminMenu::ADMIN_MENU_SLUG;
-
-		$links = [
-			\get_admin_url(null, "edit.php?post_type={$postType}"),
-			\get_admin_url(null, "edit.php?post_status=publish&post_type={$postType}"),
-			\get_admin_url(null, "edit.php?post_status=draft&post_type={$postType}"),
-			\get_admin_url(null, "edit.php?post_status=trash&post_type={$postType}"),
-			\get_admin_url(null, "edit.php?post_status=publish&post_type={$postType}"),
-			\get_admin_url(null, "edit.php?post_status=future&post_type={$postType}"),
+		$actualUrl = UtilsGeneralHelper::getCurrentUrl();
+
+		$types = [
+			Forms::POST_TYPE_SLUG,
+			Result::POST_TYPE_SLUG,
 		];
 
-		if (\in_array($actialUrl, $links, true)) {
-			echo '<script>window.location.replace("' . \get_admin_url(null, "admin.php?page={$page}") . '");</script>'; // phpcs:ignore Eightshift.Security.ComponentsEscape.OutputNotEscaped
+		foreach ($types as $type) {
+			$links = $this->getListOfLinks($type);
+
+			$typeKey = $type === Forms::POST_TYPE_SLUG ? '' : UtilsConfig::SLUG_ADMIN_LISTING_RESULTS;
```

Comment:
```suggestion
			$typeKey = ($type === Forms::POST_TYPE_SLUG) ? '' : UtilsConfig::SLUG_ADMIN_LISTING_RESULTS;
```

Just for readability sake.